package.json

```diff
@@ -2,8 +2,7 @@
   "name": "speckle-email-templates",
   "packageManager": "yarn@3.2.2",
   "scripts": {
-    "env:up": "docker-compose up -d",
-    "env:down": "docker-compose down",
+    "serve:assets": "npx http-server -p 4040",
```

Comment:
npx will install the package from scratch each time (IIRC), it's meant for one-off invocations of package binaries. if http-server is an actual dependency for the project, then just add it to `devDependencies` and convert the script to `http-server -p 4040`

Reply:
think it's cached, but legit request there!

Reply:
@didimitrie hmm, in the past I've sometimes encountered it not being cached, possibly because it installs the package using npm, but the projects deps are managed through yarn